modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating to a cluster with multi-architecture compute machines using the CLI
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on all clusters.
+* Your {product-title} version must be up to date to at least version 4.13.0. 
++
+For more information on how to update your cluster version, see _Updating a cluster using the web console_ or _Updating a cluster using the CLI_.
+* You installed the OpenShift CLI (`oc`).
+* Your `oc` client must be updated to at least verion 4.13.0. 
```

Comment:
nit: We will likely find bugs in whatever `oc` ships with 4.13.0.  They may or may not affect this particular workflow.  I thought there were `oc`/cluster skew docs somewhere, but I'm having trouble finding them now.  [This][1] has the generic:

> If you installed an earlier version of `oc`, you cannot use it to complete all of the commands in OpenShift Container Platform 4.12. Download and install the new version of `oc`.

For updates, we have so far used [the stricter][2]:

> Install the OpenShift CLI (`oc`) that matches the version for your updated version.

although if you want to follow that pattern for this case where the version number is not changing, maybe use "current cluster" or something instead of saying "updated version".

[1]: https://docs.openshift.com/container-platform/4.12/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli-web-console_cli-developer-commands
[2]: https://docs.openshift.com/container-platform/4.12/updating/updating-cluster-cli.html#update-upgrading-cli_updating-cluster-cli